app/src/components/market/sections/market_list/market_home_container.tsx

```diff
@@ -408,7 +411,9 @@ const MarketHomeContainer: React.FC = () => {
               setHasSeenBanner('true')
             }}
           >
-            <IconClose color={theme.colors.tertiary} hoverEffect size={'24'} />
+            <IconWrapper>
+              <IconClose hoverEffect size={'24'} {...restProps} />
```

Comment:
you shouldn't add restProps like this but inside the IconClose please look at the rest of icons there are couple of examples how we do it...